This is a recent commit for the new articles (#6)
apitoolkit/apitoolkit-landing: content/blog/WRITING-API-DOCUMENTATION-BEST-PRACTICES-AND-MISTAKES-TO-AVOID/_index.md

- The Quick Start Guide
The Getting Started manual gives a thorough explanation of how to use your API right now. The focus of the guide should be on helping users succeed with your API as soon as possible and providing them with support along the way. This will give readers an excellent overview of integrating and using their API.

- SDKs and Libraries
Developers can quickly call many resources thanks to code libraries. Developers will feel more at ease using your API if there are quick and simple ways to use it in several languages. SDKs are challenging to create and are not necessary for launch, but they can significantly increase API use. Having SDKs is a fantastic method to interact with the developer community if your business strategy is based on a public or open API paradigm. In such a case, there is a significant probability that if developers see value in your SDKs and APIs, they will build upon it or add more libraries. The Swagger Codegen project enables teams to quickly create SDKs from their API documentation.

- Iteractive Console

> Encourage potential customers to check what they read in the API documentation using the API console right away. A console makes getting started quick and easy, with no risk to the consumer. Experimentation is powerful. The work required to build a console or sandbox environment for users to interact with your API is rather low, but it can greatly aid engineers in understanding the value of your API graphically. Many organizations, including Microsoft and GitHub, provide interactive consoles for experimenting with their API services.

Reviewer: ```suggestion
If possible, you should also offer an interactive console to interact with your APIs. An interactive console makes getting started quick and easy, with no risk to the consumer. Encourage potential customers to test what they read in the API documentation using the API console. Experimentation is powerful. The work required to build a console or sandbox environment for users to interact with your API is rather high, but it can greatly aid engineers in understanding the value of your API graphically. Many organizations, including Microsoft and GitHub, provide interactive consoles for experimenting with their API services.
```
Show screenshots

![Mistakes-to-avoid](./WRITING-API-DOCUMENTATION-BEST-PRACTICES-AND-MISTAKES-TO-AVOID/istockphoto-1340586279-612x612.jpg)

> ## Mistakes to AVOID

Reviewer: I think you should break this Mistakes to Avoid out into a separate article.

While there are best practices there are also mistakes to avoid if you want to have only good API documentation and they include:

> - Avoid Using Jargon

Reviewer: ```suggestion
### Avoid Using too much Jargon
```

Keep in mind that you have very little control over who will use your API. Your API will be used by users with a range of skill levels, and these users will read your API documentation. You want your API and its documentation to be friendly and helpful to both expert and novice users. Writers may fall into the trap of using excessive language, which has several negative effects. The plain language will persuade people to invest in your API much more effectively than complex technical jargon. Secondly, you want your API to be helpful to as many programmers as you can, regardless of their degree of experience. If you must use technical lingo, offer a link in your API documentation to a glossary, explanation, or instructional that explains the term in question.

> - Using too many Endpoints

Reviewer: We can't tell anyone how many endpoints to have. Also, this article is about API documentation, not how they should build the APIs

Usig to many endpoints in your API will only confuse users reducing usability and readability. The ideal thing to do is to focus on developing highly flexible API  with minal endpoints that can be used to do alot of things and also increase performance. This way you get to focus on vital documentations and quick starts.

> - Focusing on Competitors

Reviewer: ```suggestion
### Focusing on Competitors
```

It doesn't have to look like that of your competitors, remember your APIs dont work alike, so its important to draw your attention back to your API and focus on how to make every client and prospect satisfiesd with what you have to offer.

> - Improper Structuring

Reviewer: ```suggestion
### Bad Documentation Structure
```

> If its not organized then its a problem, every developer wants to have easy access to what they want in order to solve their problem as fast as they can. 

Reviewer: Elaborate more on this with examples.

> Finally, a good API documentation is worthwhile even when it takes work. It is a potent tool for accelerating the development and maturity of your APIs. It lays the groundwork for an excellent developer experience when using APIs. The aforementioned advice should make it simpler for you to create quality documentation. Teams may automate the documentation process and work on creating an excellent overall experience while accessing APIs thanks to popular open source description formats like OpenAPI Specification.

Reviewer: Find a way to integrate apitoolkit into your articles.
